recipes/coin-cgl/all/conanfile.py

```diff
@@ -132,13 +133,14 @@ def package(self):
         os.unlink(os.path.join(self.package_folder, "lib", "libCgl.la"))
 
         if self.settings.compiler == "Visual Studio":
-            os.rename(os.path.join(self.package_folder, "lib", "libCgl.a"),
-                      os.path.join(self.package_folder, "lib", "Cgl.lib"))
+            rename(self,
+                   os.path.join(self.package_folder, "lib", "libCgl.a"),
+                   os.path.join(self.package_folder, "lib", "Cgl.lib"))
 
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
         self.cpp_info.libs = ["Cgl"]
-        self.cpp_info.includedirs = [os.path.join("include", "coin")]
+        self.cpp_info.includedirs.append(os.path.join("include", "coin"))
```

Comment:
What is the content of removed pkg-config file? I think includedirs was correct.

Reply:
Excerpt from cgl.pkg
```
includedir=${prefix}/include/coin
Cflags: -I${includedir}
```
Case is, other packages (coin-clp, coin-cbc, for instance) have the same behaviour (pkg-config exposes `include/{lib}`, recipe exposes both `include` and `include/{lib}`.

Reply:
Maybe these other packages are wrong. If pkgconfig only exposes include/coin, then it should be the only folder in includedirs.

Reply:
(sorry for the late answer, first weekdays are dedicated to other subjects)

Well, these recipes were added by @madebr, reviewed by @SSE4, @uilianries (and others), so there seems to lack of clear guidelines (and/or hooks?) on these points.

So we here have several options

- [ ] remove this change from the PR, other recipes left unmodified
    - pro: does not change anything
    - con: different behaviours for these recipes
- [ ] remove this change from the PR, correct other recipes
    - pro: same behaviour in all these recipes
    - con: can break consuming projects, because paths will be stripped from `includedirs`
- [ ] keep this change, other recipes left unmodified
    - pro: same behaviour in all these recipes
    - pro: "cleaner" way to include cgl headers (for those who want to)
    - con: different additional include paths compared with the original pkgconfig ones

(Note, I did not find faulty behaviour with this last option (because the current include paths are extended with a "more constrained" path), but can't be sure of it)
   
(Node, with expected guidelines on these points, I'd be happy to update these (and other) recipes, if asked for)

Reply:
if it breaks the consumers, I think it's better to keep include path